package.json

```diff
@@ -29,7 +29,7 @@
     "prepare:fiori": "cd packages/fiori && nps prepare",
     "dev:base": "cd packages/base && nps watch",
     "dev:localization": "cd packages/localization && nps watch",
-    "dev:theme-base": "cd packages/theme-base && nps watch",
+    "dev:theme-base": "cd packages/theme-base",
```

Comment:
The whole "dev:theme-base" command can be deleted both here and where it is called (in "start: all")